src/preprocess/links.rs

```diff
@@ -58,8 +59,18 @@ fn replace_all<P: AsRef<Path>>(s: &str, path: P) -> String {
 
         match playpen.render_with_path(&path) {
             Ok(new_content) => {
-                replaced.push_str(&new_content);
-                previous_end_index = playpen.end_index;
+                if depth < 200 {
```

Comment:
This can probably be pulled out into its own private `const` (e.g. `MAX_NESTED_INCLUDES_DEPTH`) so it's not a magic number. 

I highly doubt anyone's going to have more than 2 or 3 nested includes, so we can probably cap it off at 10 instead of 200. It'd be annoying if we actually overflowed the stack before hitting the 200 depth limit.